Tensile/Components/MAC_F32.py

```diff
@@ -50,6 +51,10 @@ def __call__(self, writer, m, innerUnroll):
         if not writer.asmCaps[instruction]:
             raise RuntimeError("{} instruction specified but not supported on {}".format(instruction, kernel["ISA"]))
 
+        dualMacEnable = 0
+        if writer.asmCaps["v_dual_fmac_f32"] and kernel["WavefrontSize"] == 32:
+            dualMacEnable = 1
+
```

Comment:
Is there ever a case where the instruction is available, but we don't want to use it? This change could invalidate tuning results because there is no option to generate kernels the way they were when tuning was done.